vhs-frontend/.lintstagedrc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "*.{js,jsx,ts,tsx}": "eslint --fix",
+  "*.{js,jsx,ts,tsx,css,md,json}": "prettier --write"
+}
```

Comment:
Instead of doing this in a separate config file, we can put this directly in package.json which is what we use on the projects as well.

Go here https://github.com/okonet/lint-staged and under "Configuration" read the package.json example. Convert it to that format.